src/manifests/chromium.manifest.json

```diff
@@ -18,13 +18,13 @@
   "web_accessible_resources": [
     {
       "resources": ["icons/stackOverflowBulb.svg", "icons/copy.svg"],
-			"matches": ["*stackoverflow.com/*"]
+      "matches": ["*://stackoverflow.com/*"]
```

Comment:
I think this is wrong, with ://, foreign stackoverflow versions like this one don't work.
![image](https://user-images.githubusercontent.com/52143624/209336963-8b3fcbc5-f3fa-4bbb-9b16-0ad4483054e6.png)

Reply:
This is correct for content_scripts, but not for web_accessible_resources. In fact, for Chrome `"*stackoverflow.com/*"` is an invalid matching pattern.

Reply:
Nice, I didn't know about this